README.md

```diff
@@ -217,7 +209,7 @@ The log level can be changed dynamically for adapter-instance and host (main con
 
 The states `system.adapter.xy.logLevel` and `system.host.hostname.logLevel` are updated on instance/controller start with the configured log level and can afterwards be used to change the loglevel during runtime. These changes are __not__ persisted, so the next restarts resets the loglevel to the configured one.
 
-This possibility allows to better debug adapters also during runtime.
+This possibility allows to debug adapters better also during runtime.
```

Comment:
imo this is more confusing than before

Reply:
This possibility allows to debug adapters better and during runtime.